### PR TITLE
Checkin node.properties for running `ServicesMain` locally

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
Signed-off-by: Neeharika-Sompalli [neeharika.sompalli@hedera.com](mailto:neeharika.sompalli@hedera.com)

Since `node.properties` is not checked in, it seems non trivial to understand the error. Made changes to checkin `node.properties` for devs to run `ServicesMain` locally.

### CODE REVIEW

**Review by tinker-michaelj — APPROVED** [review]
We should double- and triple-check with DevOps that the empty _node.properties_ [here](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/configuration/mainnet/node.properties) will be used during mainnet deploy, otherwise LGTM!

File: hedera-node/data/config/node.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+hedera.profiles.active=DEV
+hedera.recordStream.logDir=data/recordstreams/
+hedera.recordStream.queueCapacity=5000
+netty.mode=TEST
```
Comment (rbair23) [inline hedera-node/data/config/node.properties:4]: Let's add a newline at the end here

Comment (Neeharika-Sompalli) [reply hedera-node/data/config/node.properties]: Added ! Thanks !

**Comment by Neeharika-Sompalli** [comment]
> We should double- and triple-check with DevOps that the empty _node.properties_ [here](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/configuration/mainnet/node.properties) will be used during mainnet deploy, otherwise LGTM!

@tannerjfco @beeradb Because of this change to add `hedera-node/data/config/node.properties`, wanted to confirm that to use the empty `node.properties` for mainnet deploy as Michael mentioned.

**Review by rbair23 — APPROVED** [review]

**Comment by Neeharika-Sompalli** [comment]
> > We should double- and triple-check with DevOps that the empty _node.properties_ [here](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/configuration/mainnet/node.properties) will be used during mainnet deploy, otherwise LGTM!
> 
> @tannerjfco @beeradb Because of this change to add `hedera-node/data/config/node.properties`, wanted to confirm that to use the empty `node.properties` for mainnet deploy as Michael mentioned.

@tinker-michaelj  Confirmed with @tannerjfco  that they are using all files from `hedera-node/configuration`

**Pull request merged**